projects/angular/src/accordion/_accordion.clarity.scss

```
@@ -4,42 +4,67 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-@import './properties.accordion';
+@use 'properties.accordion';
+@use '../utils/a11y';
+@use '../utils/mixins';
```

```suggestion
@use '../utils/mixins' as *;
```
I think it would be better if we use it like that so we don't have to use it like `mixins.exports` and just do `exports` like it was.